### DISCUSSION TITLE
artalk_for_memos/

### DISCUSSION THREAD

**Discussion by giscus** [eallion/eallion.com#101] [Giscus]
# artalk_for_memos/

Memos 简介系列 《Memos 简介》 《Memos 手动导入数据》 《Memos API 调用渲染页面》 《Memos API 公告样式滚动效果》 《Memos API 获取总条数》

https://www.eallion.com/artalk_for_memos/

**Comment by thedavidweng** [comment]
Hi Charles，请问 memos v0.19.x 还有什么方法可以实现评论吗？

**Reply by eallion** [reply]
你好！
我现在也没用 Memos 了，也没怎么发现有人升级到 0.19.x 的还在用 Artalk 等评论系统的。修改比较大，过一段时间再看吧，要先等 Memos 更新稳定了可能大家再才折腾。

**Reply by thedavidweng** [reply]
感谢！我最近升级到最新版本之后感觉这个项目是不是重构/换了主要维护者，之前的 README 是中文位置，现在 Doc 和 README 都是纯英文，而且现在的 UI 也变得不那么像 Flomo 了

**Reply by eallion** [reply]
确实现在改得比较多了。可能需要再等优化一下。
